driver-core/src/test/java/com/datastax/driver/core/VirtualTableMetadataTest.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datastax.driver.core;
+
+import com.datastax.driver.core.utils.CassandraVersion;
+import org.testng.annotations.Test;
+
+import java.util.UUID;
+
+import static com.datastax.driver.core.Assertions.assertThat;
+
+@CassandraVersion("4.0.0")
+@CCMConfig()
+public class VirtualTableMetadataTest extends CCMTestsSupport {
+
+    @Test(groups = "short")
+    public void should_parse_virtual_metadata() {
```

Comment:
Looks good, just a few suggestions:

1. Inconsistent spacing between operators in some cases (i.e. no space between `=`, `>=` in some places.
2. Would be good to add comments inline for each type validated to indicate why their values are that way, i.e.:

    ``` 
    // Keyspace name should be set, marked as virtual, and have a clients table.
    // All other values should be defaulted since they are not defined in schema tables.
    ```